docs/source-app/installation_win.rst

```diff
@@ -0,0 +1,34 @@
+:orphan:
+
+#######################
+Installation on Windows
+#######################
+
+We strongly recommend to create a virtual environment first.
+Don't know what this is? Follow our `beginner guide here <install_beginner.rst>`_.
+
+Windows environments might need a little more tweaking before you install.
+
+**Requirements**
+
+* Python 3.8.x or later (3.8.x, 3.9.x, 3.10.x)
+* Pip (the latest versions of Python will already have this)
+* Git
+* PyTorch - https://pytorch.org/get-started/locally/
```

Comment:
why PyTorch is a necessity?

Reply:
I was provided the list. If this should be removed, I'll toss it.